### PR TITLE
Docs: slack links in readme and changelog. resolves #55

### LINKED ISSUE #55: List slack channel in README.md
Be great to list the newly minted slack channel in the README and possibly in the CONTRIBUTING.md file... the same applies to the other repo.

### PR DESCRIPTION
**Pull request by ZeeCoder**
I'm not sure if there's a proper format / mannerism I need to follow for this, could you advise @marcoscaceres ?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
-This spec is written using [Bikeshed](http://github.com/tabatkins/bikeshed).
-For configuration options, etc. please see the [Bikeshed docs](http://github.com/tabatkins/bikeshed).
+This spec is written using [Bikeshed](https://github.com/tabatkins/bikeshed).
+For configuration options, etc. please see the [Bikeshed docs](https://github.com/tabatkins/bikeshed).
+
+You can also contact us on [Slack](https://containerqueries.slack.com)
```
Comment (marcoscaceres) [inline CONTRIBUTING.md:4]: nit: missing full stop :)

File: README.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ The spec proper can be found at <https://wicg.github.io/cq-usecases/>.
 
 ## Found a bug? Want to contribute something?
 If you found a problem with the document, no matter how small,
-please [file a bug](https://github.com/WICG/cq-usecases/issues).
+please [file a bug](https://github.com/WICG/cq-usecases/issues)
+or get in touch with us on [Slack](https://containerqueries.slack.com)
```
Comment (marcoscaceres) [inline README.md:12]: As above.

**Review by marcoscaceres — COMMENTED** [review]
In general, this all seems fine but I honestly don't know much about Slack.

@eeeps, do folks need to explicitly invited to join the slack channel?

**Comment by marcoscaceres** [comment]
Just tested it, and yeah... looks like folks need to be explicitly invited. How does one do about doing that?

**Comment by ausi** [comment]
I think this is the correct invite-link: <https://join.slack.com/t/containerqueries/shared_invite/enQtMzA2OTc5MDUwNjk1LTEwMWEzNjcwMTY1MGYzYWMyOGMxM2MzNDM1OGZjMjM3NDNiMDMxYTk0YjQxN2FjYTZkYmZkMDZmOWE1ODRkZWI>  
@eeeps linked it here https://github.com/WICG/cq-usecases/issues/51#issuecomment-361089554

**Comment by marcoscaceres** [comment]
Won’t that link expire? If not, ✅

**Comment by eeeps** [comment]
@marcoscaceres Apparently, you can set Slack invite links to expire “never”, now. This one's set to "never", so that link should work forever.

**Comment by marcoscaceres** [comment]
@ZeeCoder, when you can, could you please update this PR with the Slack invite link? 

@eeeps, you are probably best to approve this once @ZeeCoder is done.

**Comment by ZeeCoder** [comment]
I've set up a reminder for the evening.
Sorry that I'm unresponsive, tbh I'm a bit struggling to keep up since I just moved today, and I'm also trying to prepare for my talk, among other personal things. 😅
I have a huge backlog to go through. :)

**Comment by ZeeCoder** [comment]
Closing in favour of https://github.com/WICG/cq-usecases/pull/62

**Pull request closed without merging**